src/chi-vue/src/views/SearchInputView.vue

```diff
@@ -1,74 +1,64 @@
 <template>
-  <div id="search-input-view">
-    <div class="search-input-examples__wrapper">
-      <h2>Search Input</h2>
-      <h3>xs</h3>
-      <SearchInput
-        @chiInput="(ev) => input(ev)"
-        @chiChange="(ev) => change(ev)"
-        @chiClean="() => clean()"
-        @chiFocus="() => focus()"
-        @chiBlur="() => blur()"
-        @chiSearch="(ev) => search(ev)"
-        class="-mt--2"
-        size="xs"
-      />
-      <h3>sm</h3>
-      <SearchInput class="-mt--2" size="sm" />
-      <h3>md</h3>
-      <SearchInput class="-mt--2" size="md" />
-      <h3>lg</h3>
-      <SearchInput class="-mt--2" size="lg" />
-      <h3>lg</h3>
-      <SearchInput class="-mt--2" size="xl" />
-      <h3>disabled</h3>
-      <SearchInput class="-mt--2" disabled />
-      <h3>value</h3>
-      <SearchInput class="-mt--2" value="Value" />
-      <h3>placeholder</h3>
-      <SearchInput class="-mt--2" placeholder="placeholder" />
-      <h3>name</h3>
-      <SearchInput class="-mt--2" name="name" />
-      <h3>readonly</h3>
-      <SearchInput class="-mt--2" :readOnly="true" value="Value" />
-    </div>
+  <div class="search-input-examples__wrapper">
+    <h2>Search Input</h2>
+    <h3>xs</h3>
+    <SearchInput
```

Comment:
For size xs the icon appears larger than expected:

<img width="378" alt="image" src="https://github.com/CenturyLink/Chi/assets/148064753/48591f07-3e7b-4fd4-9e50-af9b659bee85">

In documentation:

<img width="415" alt="image" src="https://github.com/CenturyLink/Chi/assets/148064753/bfce2ab8-b314-4b9f-9e6c-0dcf38325664">


I don't know if this happened before